parse.go

```diff
@@ -28,15 +28,23 @@ var dirTimeFormats = []string{
 
 // parseRFC3659ListLine parses the style of directory line defined in RFC 3659.
 func parseRFC3659ListLine(line string, now time.Time, loc *time.Location) (*Entry, error) {
+	return parseNextRFC3659ListLine(line, now, loc, &Entry{})
+}
+
+func parseNextRFC3659ListLine(line string, _ time.Time, loc *time.Location, e *Entry) (*Entry, error) {
```

Comment:
since we dont use the time and this function signature does not match `type parseFunc func(string, time.Time, *time.Location) (*Entry, error)` we should remove it:
`func parseNextRFC3659ListLine(line string, loc *time.Location, e *Entry) (*Entry, error)`

and

```
func parseRFC3659ListLine(line string, _ time.Time, loc *time.Location) (*Entry, error) {
	return parseNextRFC3659ListLine(line, loc, &Entry{})
}
```